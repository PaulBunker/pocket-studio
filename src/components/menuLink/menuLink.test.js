/* eslint-disable react/jsx-props-no-spreading */
import React from "react"
import { create } from "react-test-renderer"
import { BrowserRouter as Router } from "react-router-dom"
import MenuLink from "./menuLink"

const props = {
  text: "link text",
  target: "target",
  id: 1
}

describe.only("Testing the MenuLink component", () => {
  it("renders a link", () => {
    const link = create(
      <Router>
        <MenuLink {...props} />
      </Router>
    )
    expect(link).toMatchSnapshot()
  })
})
