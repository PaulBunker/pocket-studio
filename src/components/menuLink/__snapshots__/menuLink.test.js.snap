// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the MenuLink component renders a link 1`] = `
<a
  activeclassname="active"
  className="link"
  href="/target"
  onClick={[Function]}
>
  link text
</a>
`;
